<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplikasi Gereja</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacerTop">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>80</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="text">
       <string>&lt;h1&gt;Aplikasi Manajemen Gereja&lt;/h1&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblSub">
      <property name="text">
       <string>Gunakan menu untuk membuka form data.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacerBottom">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>260</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHALAMAN_UTAMA">
    <property name="title">
     <string>MENU</string>
    </property>
    <addaction name="actionDATA_JEMAAT"/>
    <addaction name="actionDATA_BAPTIS"/>
    <addaction name="actionJADWAL_IBADAH"/>
    <addaction name="actionPENGUMUMAN"/>
   </widget>
   <addaction name="menuHALAMAN_UTAMA"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDATA_JEMAAT">
   <property name="text">
    <string>DATA JEMAAT</string>
   </property>
  </action>
  <action name="actionDATA_BAPTIS">
   <property name="text">
    <string>DATA BAPTIS</string>
   </property>
  </action>
  <action name="actionJADWAL_IBADAH">
   <property name="text">
    <string>JADWAL IBADAH</string>
   </property>
  </action>
  <action name="actionPENGUMUMAN">
   <property name="text">
    <string>PENGUMUMAN</string>
   </property>
  </action>
  <action name="actionDATA_ADMIN">
   <property name="text">
    <string>DATA ADMIN</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
